spec/models/content_item_spec.rb

```diff
@@ -185,6 +185,18 @@
     end
   end
 
+  describe "lead_paragraph" do
```

Comment:
We add a # in front of lead_paragraph as we're describing a method:
`describe "#lead_paragraph" do`